execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -412,6 +412,12 @@ impl LmdbWasmTestBuilder {
         self
     }
 
+    /// Commit scratch to global state, and reset the scratch cache.
+    #[deprecated(since = "2.2.0", note = "renamed to `write_scratch_to_lmdb`")]
```

Comment:
As above, this function should not be deprecated - it should just replace `write_scratch_to_lmdb`.

Reply:
Done.